tools/go.mod

```diff
@@ -0,0 +1,13 @@
+module github.com/lightningnetwork/lnd/tools
+
+go 1.16
+
+require (
+	github.com/btcsuite/btcd v0.22.0-beta
```

Comment:
Please make sure you use the same versions as before here. At least `btcd` and `golangci-lint` are important to be pinned to a specific version.